pkg/scheduler/cache/cache.go

```diff
@@ -1016,30 +1014,32 @@ func (sc *SchedulerCache) processBindTask() {
 
 func (sc *SchedulerCache) BindTask() {
 	klog.V(5).Infof("batch bind task count %d", len(sc.bindCache))
-	successfulTasks := make([]*schedulingapi.TaskInfo, 0)
-	for _, task := range sc.bindCache {
-		if err := sc.VolumeBinder.BindVolumes(task, task.PodVolumes); err != nil {
-			klog.Errorf("task %s/%s bind Volumes failed: %#v", task.Namespace, task.Name, err)
-			sc.VolumeBinder.RevertVolumes(task, task.PodVolumes)
-			sc.resyncTask(task)
-		} else {
-			successfulTasks = append(successfulTasks, task)
-			klog.V(5).Infof("task %s/%s bind Volumes done", task.Namespace, task.Name)
+	go func(sc *SchedulerCache) {
+		successfulTasks := make([]*schedulingapi.TaskInfo, 0)
+		for _, task := range sc.bindCache {
+			if err := sc.VolumeBinder.BindVolumes(task, task.PodVolumes); err != nil {
+				klog.Errorf("task %s/%s bind Volumes failed: %#v", task.Namespace, task.Name, err)
+				sc.VolumeBinder.RevertVolumes(task, task.PodVolumes)
+				sc.resyncTask(task)
+			} else {
+				successfulTasks = append(successfulTasks, task)
+				klog.V(5).Infof("task %s/%s bind Volumes done", task.Namespace, task.Name)
+			}
 		}
-	}
 
-	bindTasks := make([]*schedulingapi.TaskInfo, len(successfulTasks))
-	copy(bindTasks, successfulTasks)
-	if err := sc.Bind(bindTasks); err != nil {
-		klog.Errorf("failed to bind task count %d: %#v", len(bindTasks), err)
-		return
-	}
+		bindTasks := make([]*schedulingapi.TaskInfo, len(successfulTasks))
+		copy(bindTasks, successfulTasks)
```

Comment:
Please delete the extra tabs

Reply:
ok, I should have  solved this problem 。

Reply:
> Please replace the extra tab with two spaces, which may be better，because the tab size settings vary among different editors

volcano uses tabs if you take a close look at the code🤣